recipes/nlopt/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "2.4.2" %}
+
+package:
+  name: nlopt
+  version: {{ version }}
+
+source:
+  fn: nlopt-{{ version }}.tar.gz
+  url: http://ab-initio.mit.edu/nlopt/nlopt-{{ version }}.tar.gz
+  md5: d0b8f139a4acf29b76dbae69ade8ac54
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python {{PY_VER}}*  # [not win]
+    - numpy  # [not win]
+
+  run:
+    - python {{PY_VER}}*  # [not win]
```

Comment:
@ocefpaf, msaharan I do not agree, I had to explicit {{PY_VER}}* because I encountered a bug see https://github.com/conda/conda-recipes/pull/664